WooCommerce/WooCommerceUITests/Screens/BaseScreen.swift

```diff
@@ -20,6 +20,10 @@ class BaseScreen {
     @discardableResult
     func waitForPage() throws -> BaseScreen {
         XCTContext.runActivity(named: "Confirm page \(self) is loaded") { (activity) in
```

Comment:
❤️  at the use of `XCTContext.runActivity(…)` 🙌

Reply:
That's all @rachelmcr! 😃